content/docs/render-props.md

```diff
@@ -287,8 +286,8 @@ class MouseTracker extends React.Component {
         <h1>Move the mouse around!</h1>
 
         {/*
-          This is bad! The value of the `render` prop will
-          be different on each render.
+          ¡Esto esta mal! El valor de la `render prop`
```

Comment:
```suggestion
          ¡Esto esta mal! El valor de la `render` prop
```

Reply:
I would suggest to keep it as `render prop` :thinking: